static/app/views/releases/releaseBubbles/useReleaseBubbles.tsx

```diff
@@ -128,7 +132,7 @@ function ReleaseBubbleSeries({
       return null;
     }
 
-    const numberReleases = dataItem.releases.length;
+    const numberReleases = dataItem.releases.length + dataItem.flags.length;
```

Comment:
```suggestion
    const density = dataItem.releases.length + dataItem.flags.length;
```
maybe we should rename this const here since it's no longer just the releases?